frontend/app-development/features/administration/components/News.tsx

```diff
@@ -0,0 +1,28 @@
+import { Heading, Paragraph, Label } from '@digdir/design-system-react';
+import * as React from 'react';
+
+import classes from './News.module.css';
+// import { NewsList } from 'app-shared/types/api/NewsList';
+import { useNewsListQuery } from 'app-development/hooks/queries/useNewsListQuery';
+
+export const News = () => {
+  const { data: newsList, isLoading } = useNewsListQuery();
+  if (isLoading) return null;
```

Comment:
Should we add a spinner in the middle of the NewsBox while loading and display an error if something goes wrong? 😊

Reply:
Agreed!